Auth0/ClaimValidators.swift

```diff
@@ -300,7 +300,7 @@ struct IDTokenOrgNameValidator: JWTValidator {
 
     func validate(_ jwt: JWT) -> Auth0Error? {
         guard let actualOrgName = jwt.claim(name: "org_name").string else { return ValidationError.missingOrgName }
-        guard actualOrgName.caseInsensitiveCompare(expectedOrgName) == .orderedSame else {
+        guard actualOrgName == expectedOrgName.lowercased() else {
```

Comment:
I think that may be a bug.

`caseInsensitiveCompare` will compare both strings lowercased(), so I think you need to make `actualOrgName` lowercased() as well

Reply:
@salmin-skelia do we need this fork at all?

Reply:
I don't know